packages/driver/src/cypress.ts

```diff
@@ -616,6 +616,9 @@ class $Cypress {
       case 'cy:command:end':
         return this.emit('command:end', ...args)
 
+      case 'cy:command:failed':
+        return this.emit('command:failed', ...args)
+
       case 'cy:skipped:command:end':
         return this.emit('skipped:command:end', ...args)
```

Comment:
Is it possible we need to listen to this event as well in `packages/app/src/runner/events/telemetry.ts` to end the span? I think this gets called when top changes due to a visit and we skip running any commands before the visit. They'll have a `command:start` but then this is called instead of `command:end`.

Reply:
yeah that makes sense to me

Reply:
Addressed by listening to the event